db/migrations/initial_schema.js

```diff
@@ -13,5 +13,5 @@ exports.up = knex => {
 
 exports.down = knex => {
   return knex.schema
-    .dropTableIfExists("email_hashes");
+    .dropTableIfExists("subscribers");
```

Comment:
I honestly don't know anything about knex and schemas, but is this all cool to do in the "initial_schema", or should we be doing patch files?

Not sure how much this may mess up @Micheletto when he goes to deploy to stage/prod and table names may change and we may not be dropping ye olde "email_hashes" table, etc.

Reply:
In fact, I want to make this db-altering change now, because neither stage nor prod have a database yet.

I'll manually delete the schema and run knex initial migration on heroku to test it all out before we try anything on stage/prod.